sections/sets.tex

```diff
@@ -62,7 +62,16 @@ \subsection{Finite types}
   \end{prooftree}
 \]
 
-again with the obvious equalities. The computation rule is also straightforward:
+again with the obvious equalities\footnote{
+  In the elimination rule $c$ occurs free in $E_n$; upon substituting $c$ with some $c' \in \nty{n}$, $E_n(c', \vec{d})$ yields an element of $C(c')$.
+
+  The computation rule specifies \emph{which} element.
+
+  For this pair of rules, an analogy can be drawn to typing and evaluation rule pairs in programming literature.
```

Comment:
"In programming literature" e qui non siamo in programming literature, that's about it.